src/math/Interpolate.cc

```diff
@@ -157,7 +157,7 @@ ::gsl_interp_type const *styleToGslInterpType(Interpolate::Style const style) {
         case Interpolate::UNKNOWN:
             throw LSST_EXCEPT(pex::exceptions::InvalidParameterError,
                               "I am unable to make an interpolator of type UNKNOWN");
-        case Interpolate::NUM_STYLES:
+        default:
```

Comment:
My understanding is that `default` is considered an important part of defensive programming, because even in C++ it is possible to provide illegal values (e.g. by casting).

I'm disappointed that compilers don't warn about missing values when a `default` clauses is included.